templates/json-blocks.liquid

```diff
@@ -0,0 +1,4 @@
+{% for item in json_page.blocks -%}
```

Comment:
How there handle the blocks hierarchy or our CMS designer won't support nested blocks?

Reply:
no support for nested blocks